### ISSUE TITLE
No validation failures reported using RuleFor().Custom(x=> x)

### ISSUE THREAD

**Issue by frankschierle** [JeremySkinner/FluentValidation#849]
### System Details

- FluentValidation versions:
  - 7.6.104.0
  - 8.0.0-rc1

### Issue Description
I'm currently migrating vom FluentValidation 5.6.2.0 to 7.6.104.0. In our project there are a few validators that define rules like this:

```csharp
Custom(e => new ValidationFailure(string.Empty, "ERROR"));
```

The `Custom()` method is marked as obsolete and the appropriate warning (CS0618) recommends using `RuleFor().Custom()` instead. But using this method, no errors are reported.

Example:

```csharp
class Program
{
  static void Main(string[] args)
  {
    var validator = new EntityValidator();
    var entity = new Entity();

    var result = validator.Validate(entity);
  }

  public class EntityValidator : AbstractValidator<Entity>
  {
    public EntityValidator()
    {
      this.RuleFor(e => e).Custom((e, ctx) => new ValidationFailure(string.Empty, "ERROR FROM RECOMMENDED METHOD"));
      this.Custom(e => new ValidationFailure(string.Empty, "ERROR FROM OBSOLETE METHOD"));
    }
  }

  public class Entity { }
}
```

`result` only contains the validation failure `"ERROR FROM OBSOLETE METHOD"`. The failure `"ERROR FROM RECOMMENDED METHOD"` is not included in `result`.

Version 8.0.0-rc1 is also affected.

**Comment by JeremySkinner** [comment]
Hi, your call to Custom isn’t right. You need to call ctx.AddFailure to record the custom error message. 

Please see the documentation for details: https://fluentvalidation.net/custom-validators#writing-a-custom-validator

**Comment by frankschierle** [comment]
Ok, sorry!
Don't know why I haven't seen that this is an Action, not a Func.
